### PR TITLE
diagnostics: fix various issues

### PR DESCRIPTION
**Pull request by sosiouxme**
Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1382903 and https://bugzilla.redhat.com/show_bug.cgi?id=1381611

### CODE REVIEW

**Comment by sosiouxme** [comment]
@jcantrill before i forget again to link you on this

I still have at least another thing or two I'd like to add

File: pkg/diagnostics/cluster/roles.go [hunk 1]
```diff
@@ -20,7 +20,34 @@ type ClusterRoles struct {
...
 
 const (
-	ClusterRolesName = "ClusterRoles"
+	ClusterRolesName   = "ClusterRoles"
+	clusterRoleMissing = `
+clusterrole/%s is missing.
+
+Use the 'oadm policy reconcile-cluster-roles' command to create the role. For example,
+
+  $ oadm policy reconcile-cluster-roles \
```
Comment (jcantrill) [inline pkg/diagnostics/cluster/roles.go]: For consistency we should probably update all places to include '$' in the other messages found in AGL diagnostics

Comment (sosiouxme) [inline pkg/diagnostics/cluster/roles.go]: sounds good

**Comment by sosiouxme** [comment]
[merge]

File: pkg/diagnostics/cluster/roles.go (hunk 1 above)
Comment (jcantrill) [inline pkg/diagnostics/cluster/roles.go]: I don't see where it explicitly says to not use dollar signs but I think this example is subtly different.  Here we are suggesting a command you can run to resolve a discovered issue.  We are not suggesting an example of running a diagnostics command.  Regardless, I suggest we either add '$' or remove them to be consistent with places we have used them before.

Comment (rhcarvalho) [inline pkg/diagnostics/cluster/roles.go]: > I'm not sure if it's the case here,

Most probably not the case... these seems to be just constants that I believe are printed as output of the diagnostics.

Comment (sosiouxme) [inline pkg/diagnostics/cluster/roles.go]: I could argue that bit of the CLI hacking guide is only about the command usage statement. I think diagnostics is the only place that has output related to running other commands entirely. It's a gray area; I think it is clearer with the `$` to indicate commands; what do you think?

**Comment by sosiouxme** [comment]
[merge]

**Pull request merged**